project-base/build-dev.xml

```diff
@@ -230,7 +230,6 @@
             <arg path="${path.tests}" />
             <arg path="${path.root}/*.md" />
             <arg path="${path.root}/docs" />
-            <arg value="--verbose" />
```

Comment:
This needs to be removed, because ECS 5.2+ shows name of each single file on `--verbose` option. So the output would be flooded by files.

The purpose of `-vvv`/`--debug`/`--verbose` options is to get more information, mainly in case of debugging or so, so here it should not be used.